lib/stormpath-sdk/cache/memcached_store.rb

```diff
@@ -3,34 +3,40 @@
 module Stormpath
   module Cache
     class MemcachedStore
+      attr_reader :memcached, :options
+
       def initialize(opts = {})
-        options = nil if opts.blank?
-        @memcached = Memcached.new(options)
+        @options = opts.blank? ? { host: 'localhost:11211' } : opts
```

Comment:
found it. Looks good. I would only move the default host definition into a constant.

```ruby
class MemcachedStore
  DEFAULT_SERVER_HOST = 'localhost:11211'
end
```

in initialize
```ruby
options = opts.blank? ? { host: DEFAULT_SERVER_HOST } : opts
```